library/components/TTooltip.vue

```diff
@@ -83,27 +80,36 @@ export default {
 			this.positions = positions;
 		},
 		handleMouseOver() {
-			if (this.isFocused) return;
+			// Mouseover should also prepare position of tooltip,
+			// Without $nextTick, behaviour is glitched.
+			this.$nextTick(() => {
+				this.placeTooltip();
+			})
+
+			if (this.open) {
+				return;
+			}
 			this.show();
 		},
 		handleMouseLeave() {
-			if (this.isFocused) return;
+			if (this.open) {
+				return;
+			}
 			this.hide();
 		},
-		handleClick() {
-            this.isFocused = true;
-            this.show();
-		},
 		show() {
-			this.showTooltip = true;
-			// Glitchy behaviour without nextTick()
-			this.$nextTick(() => {
-				this.placeTooltip();
-			});
+			this.isVisible = true;
 		},
 		hide() {
-			this.showTooltip = false;
-            this.isFocused = false;
+			this.isVisible = false;
+		}
+	},
+	watch: {
+		open(isOpen) {
```

Comment:
I couldn't see from where this `isOpen` comes from

Reply:
You the parameter? the prop `open` is being watched and it's value `isOpen` can be true or false.